### PR TITLE
Add Belgian battery capacity

### PR DESCRIPTION
**Pull request by kroeckx**
Fixes: #6339

### Double check

- [ ] I have tested my parser changes locally with `poetry run test_parser "zone_key"`
- [ ] I have run `pnpx prettier --write .` and `poetry run format` to format my changes.

I have no idea if "battery: " is correct syntax. I suspect it's not, since I can't find any examples with that and that there is currently just no way to specify it's the power output, only the the storage capacity.

### CODE REVIEW

File: config/zones/BE.yaml [hunk 1]
```diff
@@ -15,6 +15,8 @@ capacity:
   oil: 455
   solar: 8595
   wind: 5315
+  battery: 50
+  battery storage: 100
```
Comment (VIKTORVAV99) [inline config/zones/BE.yaml:19]:
```suggestion
  battery storage: 50
```
We don't support the storage values and only the discharge value should be added, at least for now.

We also need to add the source to `DATA_SOURCES.md` before we can merge this.

**Comment by kroeckx** [comment]
I've pushed a new version.

File: DATA_SOURCES.md [hunk 2]
```diff
@@ -198,6 +198,7 @@ For many European countries, data is available from [ENTSO-E](https://transparen
   - Solar: [ELIA](https://www.elia.be/en/grid-data/power-generation/solar-pv-power-generation-data)
   - Hydro: [ELIA](https://www.elia.be/-/media/project/elia/shared/documents/elia-group/publications-pdfs/20200414_elia_annual-report-sustainability_en.pdf)
   - Wind: [ELIA](https://www.elia.be/en/grid-data/power-generation/wind-power-generation)
+  - Battery: [ELIA](https://www.elia.be/en/publications/reports)
```
Comment (VIKTORVAV99) [inline DATA_SOURCES.md:201]: It would be great if we could specify which report it was here. They seem to have quite a few 😅

**Comment by kroeckx** [comment]
It's on that page itself.

**Review by VIKTORVAV99 — APPROVED** [review]
LGTM! 🎉

Thanks for adding it!

**Pull request merged**